sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -363,11 +363,7 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
     /**
      * @brief HTTP retry policy.
      */
-    class RetryPolicy
-#if !defined(_azure_TESTING_BUILD)
-        final
-#endif
-        : public HttpPolicy {
+    class RetryPolicy : public HttpPolicy {
```

Comment:
I would like to see the design of how we intend to inherit from this to solve the underlying storage feature, **first**, before merging this change.

Reply:
I updated the PR.